Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -364,7 +364,6 @@ public override void Analyze(AnalyzeContext context)
 
                 case ValidationState.Unknown:
                 {
-                    level = FailureLevel.Note;
```

Comment:
>level = FailureLevel.Note; [](http://example.com/codeflow?start=20&length=26)

Removing the overwriting of the failure level if only static validation has completed. This is the actual code change for this PR.

Reply:
Ah, but your change is incomplete, I think. You found the reason that this data was getting clobbered. You did *not*, however, distinguish handling between validators that do and do *not* have a dynamic validator and my supposition is that these cases aren't identical.